src/main/java/com/minecolonies/core/entity/citizen/EntityCitizen.java

```diff
@@ -1944,14 +1945,16 @@ public void refreshDimensions()
     @Override
     public void queueSound(@NotNull final SoundEvent soundEvent, final BlockPos pos, final int length, final int repetitions)
     {
-        Network.getNetwork().sendToTrackingEntity(new PlaySoundForCitizenMessage(this.getId(), soundEvent, this.getSoundSource(), pos, level, length, repetitions), this);
+        Network.getNetwork().sendToPosition(new PlaySoundForCitizenMessage(this.getId(), soundEvent, this.getSoundSource(), pos, level, length, repetitions),
+          new PacketDistributor.TargetPoint(pos.getX(), pos.getY(), pos.getZ(), BLOCK_BREAK_SOUND_RANGE, level.dimension()));
```

Comment:
Not sure if this makes any difference? the previous sends it to all players who can see/are near the entity, the new one sends it to all players who are near the entity position

Reply:
I think entity tracking goes much much farther than pos tracking. Like we see entities much farther away.

Reply:
nah pos tracking is further, its the chunks you see around you, entity is closer somewhat

Reply:
Using BLOCK_BREAK_SOUND_RANGE for 16 blocks range

Reply:
ye that is shorter then